diff --git a/node_modules/@shopify/flash-list/CHANGELOG.md b/node_modules/@shopify/flash-list/CHANGELOG.md
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/LICENSE.md b/node_modules/@shopify/flash-list/LICENSE.md
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/README.md b/node_modules/@shopify/flash-list/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/RNFlashList.podspec b/node_modules/@shopify/flash-list/RNFlashList.podspec
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/build.gradle b/node_modules/@shopify/flash-list/android/build.gradle
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/AndroidManifest.xml b/node_modules/@shopify/flash-list/android/src/main/AndroidManifest.xml
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutShadow.kt b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutShadow.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutView.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutViewManager.kt b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/AutoLayoutViewManager.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/CellContainer.java b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/CellContainer.java
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/CellContainerImpl.kt b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/CellContainerImpl.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/CellContainerManager.kt b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/CellContainerManager.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/FlashListPackage.kt b/node_modules/@shopify/flash-list/android/src/main/kotlin/com/shopify/reactnative/flash_list/FlashListPackage.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/AutoLayoutShadowTest.kt b/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/AutoLayoutShadowTest.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/models/Rect.kt b/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/models/Rect.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/models/TestCollection.kt b/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/models/TestCollection.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/models/TestDataModel.kt b/node_modules/@shopify/flash-list/android/src/test/java/com/shopify/reactnative/flash_list/models/TestDataModel.kt
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/android/src/test/resources/LayoutTestData.json b/node_modules/@shopify/flash-list/android/src/test/resources/LayoutTestData.json
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/AnimatedFlashList.d.ts b/node_modules/@shopify/flash-list/dist/AnimatedFlashList.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/AnimatedFlashList.d.ts.map b/node_modules/@shopify/flash-list/dist/AnimatedFlashList.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/AnimatedFlashList.js b/node_modules/@shopify/flash-list/dist/AnimatedFlashList.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/AnimatedFlashList.js.map b/node_modules/@shopify/flash-list/dist/AnimatedFlashList.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashList.d.ts b/node_modules/@shopify/flash-list/dist/FlashList.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashList.d.ts.map b/node_modules/@shopify/flash-list/dist/FlashList.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashList.js b/node_modules/@shopify/flash-list/dist/FlashList.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashList.js.map b/node_modules/@shopify/flash-list/dist/FlashList.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashListProps.d.ts b/node_modules/@shopify/flash-list/dist/FlashListProps.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashListProps.d.ts.map b/node_modules/@shopify/flash-list/dist/FlashListProps.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashListProps.js b/node_modules/@shopify/flash-list/dist/FlashListProps.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/FlashListProps.js.map b/node_modules/@shopify/flash-list/dist/FlashListProps.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.d.ts b/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.d.ts.map b/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.js b/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.js.map b/node_modules/@shopify/flash-list/dist/GridLayoutProviderWithProps.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts b/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts
old mode 100644
new mode 100755
index b328c8c..3dd2e14
--- a/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts
+++ b/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts
@@ -1,51 +1 @@
-import React from "react";
-import { NativeScrollEvent } from "react-native";
-import FlashList from "./FlashList";
-import { FlashListProps, ListRenderItemInfo } from "./FlashListProps";
-export interface MasonryListRenderItemInfo<TItem> extends ListRenderItemInfo<TItem> {
-    columnSpan: number;
-    columnIndex: number;
-}
-export declare type MasonryListRenderItem<TItem> = (info: MasonryListRenderItemInfo<TItem>) => React.ReactElement | null;
-export interface MasonryFlashListProps<T> extends Omit<FlashListProps<T>, "horizontal" | "initialScrollIndex" | "inverted" | "onBlankArea" | "renderItem" | "viewabilityConfigCallbackPairs"> {
-    /**
-     * Allows you to change the column widths of the list. This is helpful if you want some columns to be wider than the others.
-     * e.g, if `numColumns` is `3`, you can return `2` for `index 1` and `1` for the rest to achieve a `1:2:1` split by width.
-     */
-    getColumnFlex?: (items: MasonryListItem<T>[], columnIndex: number, maxColumns: number, extraData?: any) => number;
-    /**
-     * If enabled, MasonryFlashList will try to reduce difference in column height by modifying item order.
-     * `overrideItemLayout` is required to make this work.
-     */
-    optimizeItemArrangement?: boolean;
-    /**
-     * Extends typical `renderItem` to include `columnIndex` and `columnSpan` (number of columns the item spans).
-     * `columnIndex` gives the consumer column information in case they might need to treat items differently based on column.
-     * This information may not otherwise be derived if using the `optimizeItemArrangement` feature, as the items will no
-     * longer be linearly distributed across the columns; instead they are allocated to the column with the least estimated height.
-     */
-    renderItem: MasonryListRenderItem<T> | null | undefined;
-}
-export interface MasonryFlashListScrollEvent extends NativeScrollEvent {
-    doNotPropagate?: boolean;
-}
-export interface MasonryListItem<T> {
-    originalIndex: number;
-    originalItem: T;
-}
-/**
- * MasonryFlashListRef with support for scroll related methods
- */
-export interface MasonryFlashListRef<T> {
-    scrollToOffset: FlashList<T>["scrollToOffset"];
-    scrollToEnd: FlashList<T>["scrollToEnd"];
-    getScrollableNode: FlashList<T>["getScrollableNode"];
-}
-/**
- * FlashList variant that enables rendering of masonry layouts.
- * If you want `MasonryFlashList` to optimize item arrangement, enable `optimizeItemArrangement` and pass a valid `overrideItemLayout` function.
- */
-export declare const MasonryFlashList: <T>(props: MasonryFlashListProps<T> & {
-    ref?: React.RefObject<MasonryFlashListRef<T>> | undefined;
-}) => React.ReactElement;
-//# sourceMappingURL=MasonryFlashList.d.ts.map
\ No newline at end of file
+export { FlashList as MasonryFlashList } from './FlashList';
diff --git a/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts.map b/node_modules/@shopify/flash-list/dist/MasonryFlashList.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/MasonryFlashList.js b/node_modules/@shopify/flash-list/dist/MasonryFlashList.js
old mode 100644
new mode 100755
index 5d43fd7..f149856
--- a/node_modules/@shopify/flash-list/dist/MasonryFlashList.js
+++ b/node_modules/@shopify/flash-list/dist/MasonryFlashList.js
@@ -1,250 +1,5 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.MasonryFlashList = void 0;
-var tslib_1 = require("tslib");
-var react_1 = tslib_1.__importStar(require("react"));
-var react_native_1 = require("react-native");
-var CustomError_1 = tslib_1.__importDefault(require("./errors/CustomError"));
-var ExceptionList_1 = tslib_1.__importDefault(require("./errors/ExceptionList"));
-var FlashList_1 = tslib_1.__importDefault(require("./FlashList"));
-var ContentContainerUtils_1 = require("./utils/ContentContainerUtils");
-var defaultEstimatedItemSize = 100;
-/**
- * FlashList variant that enables rendering of masonry layouts.
- * If you want `MasonryFlashList` to optimize item arrangement, enable `optimizeItemArrangement` and pass a valid `overrideItemLayout` function.
- */
-var MasonryFlashListComponent = react_1.default.forwardRef(function (
-/**
- * Forward Ref will force cast generic parament T to unknown. Export has a explicit cast to solve this.
- */
-props, forwardRef) {
-    var _a, _b, _c, _d, _e;
-    var columnCount = props.numColumns || 1;
-    var drawDistance = props.drawDistance;
-    var estimatedListSize = (_b = (_a = props.estimatedListSize) !== null && _a !== void 0 ? _a : react_native_1.Dimensions.get("window")) !== null && _b !== void 0 ? _b : { height: 500, width: 500 };
-    if (props.optimizeItemArrangement && !props.overrideItemLayout) {
-        throw new CustomError_1.default(ExceptionList_1.default.overrideItemLayoutRequiredForMasonryOptimization);
-    }
-    var dataSet = useDataSet(columnCount, Boolean(props.optimizeItemArrangement), props.data, props.overrideItemLayout, props.extraData);
-    var totalColumnFlex = useTotalColumnFlex(dataSet, props);
-    var onScrollRef = (0, react_1.useRef)([]);
-    var emptyScrollEvent = (0, react_1.useRef)(getEmptyScrollEvent())
-        .current;
-    var ScrollComponent = (0, react_1.useRef)(getFlashListScrollView(onScrollRef, function () {
-        var _a;
-        return (((_a = getListRenderedSize(parentFlashList)) === null || _a === void 0 ? void 0 : _a.height) ||
-            estimatedListSize.height);
-    })).current;
-    var onScrollProxy = (0, react_1.useRef)(function (scrollEvent) {
-        var _a, _b, _c, _d;
-        emptyScrollEvent.nativeEvent.contentOffset.y =
-            scrollEvent.nativeEvent.contentOffset.y -
-                ((_b = (_a = parentFlashList.current) === null || _a === void 0 ? void 0 : _a.firstItemOffset) !== null && _b !== void 0 ? _b : 0);
-        (_c = onScrollRef.current) === null || _c === void 0 ? void 0 : _c.forEach(function (onScrollCallback) {
-            onScrollCallback === null || onScrollCallback === void 0 ? void 0 : onScrollCallback(emptyScrollEvent);
-        });
-        if (!scrollEvent.nativeEvent.doNotPropagate) {
-            (_d = props.onScroll) === null || _d === void 0 ? void 0 : _d.call(props, scrollEvent);
-        }
-    }).current;
-    /**
-     * We're triggering an onScroll on internal lists so that they register the correct offset which is offset - header size.
-     * This will make sure viewability callbacks are triggered correctly.
-     * 32 ms is equal to two frames at 60 fps. Faster framerates will not cause any problems.
-     */
-    var onLoadForNestedLists = (0, react_1.useRef)(function (args) {
-        var _a;
-        setTimeout(function () {
-            emptyScrollEvent.nativeEvent.doNotPropagate = true;
-            onScrollProxy === null || onScrollProxy === void 0 ? void 0 : onScrollProxy(emptyScrollEvent);
-            emptyScrollEvent.nativeEvent.doNotPropagate = false;
-        }, 32);
-        (_a = props.onLoad) === null || _a === void 0 ? void 0 : _a.call(props, args);
-    }).current;
-    var _f = tslib_1.__read(useRefWithForwardRef(forwardRef), 2), parentFlashList = _f[0], getFlashList = _f[1];
-    var renderItem = props.renderItem, getItemType = props.getItemType, getColumnFlex = props.getColumnFlex, overrideItemLayout = props.overrideItemLayout, viewabilityConfig = props.viewabilityConfig, keyExtractor = props.keyExtractor, onLoad = props.onLoad, onViewableItemsChanged = props.onViewableItemsChanged, data = props.data, stickyHeaderIndices = props.stickyHeaderIndices, CellRendererComponent = props.CellRendererComponent, ItemSeparatorComponent = props.ItemSeparatorComponent, remainingProps = tslib_1.__rest(props, ["renderItem", "getItemType", "getColumnFlex", "overrideItemLayout", "viewabilityConfig", "keyExtractor", "onLoad", "onViewableItemsChanged", "data", "stickyHeaderIndices", "CellRendererComponent", "ItemSeparatorComponent"]);
-    var firstColumnHeight = ((_d = (_c = dataSet[0]) === null || _c === void 0 ? void 0 : _c.length) !== null && _d !== void 0 ? _d : 0) *
-        ((_e = props.estimatedItemSize) !== null && _e !== void 0 ? _e : defaultEstimatedItemSize);
-    var insetForLayoutManager = (0, ContentContainerUtils_1.applyContentContainerInsetForLayoutManager)({ height: 0, width: 0 }, props.contentContainerStyle, false);
-    return (react_1.default.createElement(FlashList_1.default, tslib_1.__assign({ ref: getFlashList }, remainingProps, { horizontal: false, numColumns: columnCount, data: dataSet, onScroll: onScrollProxy, estimatedItemSize: firstColumnHeight || estimatedListSize.height, renderItem: function (args) {
-            var _a, _b;
-            return (react_1.default.createElement(FlashList_1.default, { renderScrollComponent: ScrollComponent, estimatedItemSize: props.estimatedItemSize, data: args.item, onLoad: args.index === 0 ? onLoadForNestedLists : undefined, renderItem: function (innerArgs) {
-                    var _a;
-                    return ((_a = renderItem === null || renderItem === void 0 ? void 0 : renderItem(tslib_1.__assign(tslib_1.__assign({}, innerArgs), { item: innerArgs.item.originalItem, index: innerArgs.item.originalIndex, columnSpan: 1, columnIndex: args.index }))) !== null && _a !== void 0 ? _a : null);
-                }, keyExtractor: keyExtractor
-                    ? function (item, _) {
-                        return keyExtractor === null || keyExtractor === void 0 ? void 0 : keyExtractor(item.originalItem, item.originalIndex);
-                    }
-                    : undefined, getItemType: getItemType
-                    ? function (item, _, extraData) {
-                        return getItemType === null || getItemType === void 0 ? void 0 : getItemType(item.originalItem, item.originalIndex, extraData);
-                    }
-                    : undefined, drawDistance: drawDistance, estimatedListSize: {
-                    height: estimatedListSize.height,
-                    width: (((((_a = getListRenderedSize(parentFlashList)) === null || _a === void 0 ? void 0 : _a.width) ||
-                        estimatedListSize.width) +
-                        insetForLayoutManager.width) /
-                        totalColumnFlex) *
-                        ((_b = getColumnFlex === null || getColumnFlex === void 0 ? void 0 : getColumnFlex(args.item, args.index, columnCount, props.extraData)) !== null && _b !== void 0 ? _b : 1),
-                }, extraData: props.extraData, CellRendererComponent: CellRendererComponent, ItemSeparatorComponent: ItemSeparatorComponent, viewabilityConfig: viewabilityConfig, onViewableItemsChanged: onViewableItemsChanged
-                    ? function (info) {
-                        updateViewTokens(info.viewableItems);
-                        updateViewTokens(info.changed);
-                        onViewableItemsChanged === null || onViewableItemsChanged === void 0 ? void 0 : onViewableItemsChanged(info);
-                    }
-                    : undefined, overrideItemLayout: overrideItemLayout
-                    ? function (layout, item, _, __, extraData) {
-                        overrideItemLayout === null || overrideItemLayout === void 0 ? void 0 : overrideItemLayout(layout, item.originalItem, item.originalIndex, columnCount, extraData);
-                        layout.span = undefined;
-                    }
-                    : undefined }));
-        }, overrideItemLayout: getColumnFlex
-            ? function (layout, item, index, maxColumns, extraData) {
-                layout.span =
-                    (columnCount *
-                        getColumnFlex(item, index, maxColumns, extraData)) /
-                        totalColumnFlex;
-            }
-            : undefined })));
-});
-/**
- * Splits data for each column's FlashList
- */
-var useDataSet = function (columnCount, optimizeItemArrangement, sourceData, overrideItemLayout, extraData) {
-    return (0, react_1.useMemo)(function () {
-        var _a;
-        if (!sourceData || sourceData.length === 0) {
-            return [];
-        }
-        var columnHeightTracker = new Array(columnCount).fill(0);
-        var layoutObject = { size: undefined };
-        var dataSet = new Array(columnCount);
-        var dataSize = sourceData.length;
-        for (var i = 0; i < columnCount; i++) {
-            dataSet[i] = [];
-        }
-        for (var i = 0; i < dataSize; i++) {
-            var nextColumnIndex = i % columnCount;
-            if (optimizeItemArrangement) {
-                for (var j = 0; j < columnCount; j++) {
-                    if (columnHeightTracker[j] < columnHeightTracker[nextColumnIndex]) {
-                        nextColumnIndex = j;
-                    }
-                }
-                // update height of column
-                layoutObject.size = undefined;
-                overrideItemLayout(layoutObject, sourceData[i], i, columnCount, extraData);
-                columnHeightTracker[nextColumnIndex] +=
-                    (_a = layoutObject.size) !== null && _a !== void 0 ? _a : defaultEstimatedItemSize;
-            }
-            dataSet[nextColumnIndex].push({
-                originalItem: sourceData[i],
-                originalIndex: i,
-            });
-        }
-        return dataSet;
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [sourceData, columnCount, optimizeItemArrangement, extraData]);
-};
-var useTotalColumnFlex = function (dataSet, props) {
-    return (0, react_1.useMemo)(function () {
-        var columnCount = props.numColumns || 1;
-        if (!props.getColumnFlex) {
-            return columnCount;
-        }
-        var totalFlexSum = 0;
-        var dataSize = dataSet.length;
-        for (var i = 0; i < dataSize; i++) {
-            totalFlexSum += props.getColumnFlex(dataSet[i], i, columnCount, props.extraData);
-        }
-        return totalFlexSum;
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [dataSet, props.getColumnFlex, props.extraData]);
-};
-/**
- * Handle both function refs and refs with current property
- */
-var useRefWithForwardRef = function (forwardRef) {
-    var ref = (0, react_1.useRef)(null);
-    return [
-        ref,
-        (0, react_1.useCallback)(function (instance) {
-            ref.current = instance;
-            if (typeof forwardRef === "function") {
-                forwardRef(instance);
-            }
-            else if (forwardRef) {
-                forwardRef.current = instance;
-            }
-        }, [forwardRef]),
-    ];
-};
-/**
- * This ScrollView is actually just a view mimicking a scrollview. We block the onScroll event from being passed to the parent list directly.
- * We manually drive onScroll from the parent and thus, achieve recycling.
- */
-var getFlashListScrollView = function (onScrollRef, getParentHeight) {
-    var FlashListScrollView = react_1.default.forwardRef(function (props, ref) {
-        var onLayout = props.onLayout, onScroll = props.onScroll, rest = tslib_1.__rest(props, ["onLayout", "onScroll"]);
-        var onLayoutProxy = (0, react_1.useCallback)(function (layoutEvent) {
-            onLayout === null || onLayout === void 0 ? void 0 : onLayout({
-                nativeEvent: {
-                    layout: {
-                        height: getParentHeight(),
-                        width: layoutEvent.nativeEvent.layout.width,
-                    },
-                },
-            });
-        }, [onLayout]);
-        (0, react_1.useEffect)(function () {
-            var _a;
-            if (onScroll) {
-                (_a = onScrollRef.current) === null || _a === void 0 ? void 0 : _a.push(onScroll);
-            }
-            return function () {
-                if (!onScrollRef.current || !onScroll) {
-                    return;
-                }
-                var indexToDelete = onScrollRef.current.indexOf(onScroll);
-                if (indexToDelete > -1) {
-                    onScrollRef.current.splice(indexToDelete, 1);
-                }
-            };
-        }, [onScroll]);
-        return react_1.default.createElement(react_native_1.View, tslib_1.__assign({ ref: ref }, rest, { onLayout: onLayoutProxy }));
-    });
-    FlashListScrollView.displayName = "FlashListScrollView";
-    return FlashListScrollView;
-};
-var updateViewTokens = function (tokens) {
-    var length = tokens.length;
-    for (var i = 0; i < length; i++) {
-        var token = tokens[i];
-        if (token.index !== null && token.index !== undefined) {
-            if (token.item) {
-                token.index = token.item.originalIndex;
-                token.item = token.item.originalItem;
-            }
-            else {
-                token.index = null;
-                token.item = undefined;
-            }
-        }
-    }
-};
-var getEmptyScrollEvent = function () {
-    return {
-        nativeEvent: { contentOffset: { y: 0, x: 0 } },
-    };
-};
-var getListRenderedSize = function (parentFlashList) {
-    var _a, _b;
-    return (_b = (_a = parentFlashList === null || parentFlashList === void 0 ? void 0 : parentFlashList.current) === null || _a === void 0 ? void 0 : _a.recyclerlistview_unsafe) === null || _b === void 0 ? void 0 : _b.getRenderedSize();
-};
-MasonryFlashListComponent.displayName = "MasonryFlashList";
-/**
- * FlashList variant that enables rendering of masonry layouts.
- * If you want `MasonryFlashList` to optimize item arrangement, enable `optimizeItemArrangement` and pass a valid `overrideItemLayout` function.
- */
-exports.MasonryFlashList = MasonryFlashListComponent;
-//# sourceMappingURL=MasonryFlashList.js.map
\ No newline at end of file
+var FlashList_1 = require("./FlashList");
+exports.MasonryFlashList = FlashList_1.FlashList;
diff --git a/node_modules/@shopify/flash-list/dist/MasonryFlashList.js.map b/node_modules/@shopify/flash-list/dist/MasonryFlashList.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/PureComponentWrapper.d.ts b/node_modules/@shopify/flash-list/dist/PureComponentWrapper.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/PureComponentWrapper.d.ts.map b/node_modules/@shopify/flash-list/dist/PureComponentWrapper.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/PureComponentWrapper.js b/node_modules/@shopify/flash-list/dist/PureComponentWrapper.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/PureComponentWrapper.js.map b/node_modules/@shopify/flash-list/dist/PureComponentWrapper.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.js b/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/AverageWindow.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.js b/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/ContentContainerUtils.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.js b/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/FlashList.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.js b/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/GridLayoutProviderWithProps.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.js b/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/MasonryFlashList.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.js b/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/PlatformHelper.web.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.js b/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/ViewabilityHelper.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.js b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.js.map b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountFlashList.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.js b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.js.map b/node_modules/@shopify/flash-list/dist/__tests__/helpers/mountMasonryFlashList.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.d.ts b/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.d.ts.map b/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.js b/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.js.map b/node_modules/@shopify/flash-list/dist/__tests__/useBlankAreaTracker.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.d.ts b/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.d.ts.map b/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.js b/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.js.map b/node_modules/@shopify/flash-list/dist/benchmark/AutoScrollHelper.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.d.ts b/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.d.ts.map b/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.js b/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.js.map b/node_modules/@shopify/flash-list/dist/benchmark/JSFPSMonitor.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.d.ts b/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.d.ts.map b/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.js b/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.js.map b/node_modules/@shopify/flash-list/dist/benchmark/roundToDecimalPlaces.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.d.ts b/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.d.ts.map b/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.js b/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.js.map b/node_modules/@shopify/flash-list/dist/benchmark/useBenchmark.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.d.ts b/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.d.ts.map b/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.js b/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.js.map b/node_modules/@shopify/flash-list/dist/benchmark/useBlankAreaTracker.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.d.ts b/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.d.ts.map b/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.js b/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.js.map b/node_modules/@shopify/flash-list/dist/benchmark/useDataMultiplier.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.d.ts b/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.d.ts.map b/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.js b/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.js.map b/node_modules/@shopify/flash-list/dist/benchmark/useFlatListBenchmark.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/CustomError.d.ts b/node_modules/@shopify/flash-list/dist/errors/CustomError.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/CustomError.d.ts.map b/node_modules/@shopify/flash-list/dist/errors/CustomError.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/CustomError.js b/node_modules/@shopify/flash-list/dist/errors/CustomError.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/CustomError.js.map b/node_modules/@shopify/flash-list/dist/errors/CustomError.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/ExceptionList.d.ts b/node_modules/@shopify/flash-list/dist/errors/ExceptionList.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/ExceptionList.d.ts.map b/node_modules/@shopify/flash-list/dist/errors/ExceptionList.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/ExceptionList.js b/node_modules/@shopify/flash-list/dist/errors/ExceptionList.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/ExceptionList.js.map b/node_modules/@shopify/flash-list/dist/errors/ExceptionList.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/Warnings.d.ts b/node_modules/@shopify/flash-list/dist/errors/Warnings.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/Warnings.d.ts.map b/node_modules/@shopify/flash-list/dist/errors/Warnings.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/Warnings.js b/node_modules/@shopify/flash-list/dist/errors/Warnings.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/errors/Warnings.js.map b/node_modules/@shopify/flash-list/dist/errors/Warnings.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/index.d.ts b/node_modules/@shopify/flash-list/dist/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/index.d.ts.map b/node_modules/@shopify/flash-list/dist/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/index.js b/node_modules/@shopify/flash-list/dist/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/index.js.map b/node_modules/@shopify/flash-list/dist/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.d.ts b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.d.ts.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.js b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.js.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutView.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.d.ts b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.d.ts.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.js b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.js.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.android.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.d.ts b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.d.ts.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.d.ts b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.d.ts.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.js b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.js.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.ios.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.js b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.js.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponent.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.d.ts b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.d.ts.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.js b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.js.map b/node_modules/@shopify/flash-list/dist/native/auto-layout/AutoLayoutViewNativeComponentProps.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.d.ts b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.d.ts.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.js b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.js.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.android.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.d.ts b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.d.ts.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.d.ts b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.d.ts.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.js b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.js.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.ios.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.js b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.js.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.d.ts b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.d.ts.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.js b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.js.map b/node_modules/@shopify/flash-list/dist/native/cell-container/CellContainer.web.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.d.ts b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.d.ts.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.js b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.js.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.android.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.d.ts b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.d.ts.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.d.ts b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.d.ts.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.js b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.js.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.ios.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.js b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.js.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.d.ts b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.d.ts.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.js b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.js.map b/node_modules/@shopify/flash-list/dist/native/config/PlatformHelper.web.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/tsconfig.tsbuildinfo b/node_modules/@shopify/flash-list/dist/tsconfig.tsbuildinfo
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/AverageWindow.d.ts b/node_modules/@shopify/flash-list/dist/utils/AverageWindow.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/AverageWindow.d.ts.map b/node_modules/@shopify/flash-list/dist/utils/AverageWindow.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/AverageWindow.js b/node_modules/@shopify/flash-list/dist/utils/AverageWindow.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/AverageWindow.js.map b/node_modules/@shopify/flash-list/dist/utils/AverageWindow.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.d.ts b/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.d.ts.map b/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.js b/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.js.map b/node_modules/@shopify/flash-list/dist/utils/ContentContainerUtils.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewToken.d.ts b/node_modules/@shopify/flash-list/dist/viewability/ViewToken.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewToken.d.ts.map b/node_modules/@shopify/flash-list/dist/viewability/ViewToken.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewToken.js b/node_modules/@shopify/flash-list/dist/viewability/ViewToken.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewToken.js.map b/node_modules/@shopify/flash-list/dist/viewability/ViewToken.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.d.ts b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.d.ts.map b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.js b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.js.map b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityHelper.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.d.ts b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.d.ts.map b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.js b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.js.map b/node_modules/@shopify/flash-list/dist/viewability/ViewabilityManager.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/RNFlashList.xcodeproj/project.pbxproj b/node_modules/@shopify/flash-list/ios/RNFlashList.xcodeproj/project.pbxproj
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/RNFlashList.xcodeproj/project.xcworkspace/contents.xcworkspacedata b/node_modules/@shopify/flash-list/ios/RNFlashList.xcodeproj/project.xcworkspace/contents.xcworkspacedata
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Sources/AutoLayoutView.swift b/node_modules/@shopify/flash-list/ios/Sources/AutoLayoutView.swift
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Sources/AutoLayoutViewManager.m b/node_modules/@shopify/flash-list/ios/Sources/AutoLayoutViewManager.m
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Sources/AutoLayoutViewManager.swift b/node_modules/@shopify/flash-list/ios/Sources/AutoLayoutViewManager.swift
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Sources/CellContainer.swift b/node_modules/@shopify/flash-list/ios/Sources/CellContainer.swift
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Sources/CellContainerManager.m b/node_modules/@shopify/flash-list/ios/Sources/CellContainerManager.m
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Sources/CellContainerManager.swift b/node_modules/@shopify/flash-list/ios/Sources/CellContainerManager.swift
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Sources/FlatListPro-Bridging-Header.h b/node_modules/@shopify/flash-list/ios/Sources/FlatListPro-Bridging-Header.h
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/ios/Tests/AutoLayoutViewTests.swift b/node_modules/@shopify/flash-list/ios/Tests/AutoLayoutViewTests.swift
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/jestSetup.js b/node_modules/@shopify/flash-list/jestSetup.js
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/AnimatedFlashList.ts b/node_modules/@shopify/flash-list/src/AnimatedFlashList.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/FlashList.tsx b/node_modules/@shopify/flash-list/src/FlashList.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/FlashListProps.ts b/node_modules/@shopify/flash-list/src/FlashListProps.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/GridLayoutProviderWithProps.ts b/node_modules/@shopify/flash-list/src/GridLayoutProviderWithProps.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/MasonryFlashList.tsx b/node_modules/@shopify/flash-list/src/MasonryFlashList.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/PureComponentWrapper.tsx b/node_modules/@shopify/flash-list/src/PureComponentWrapper.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/AverageWindow.test.ts b/node_modules/@shopify/flash-list/src/__tests__/AverageWindow.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/ContentContainerUtils.test.ts b/node_modules/@shopify/flash-list/src/__tests__/ContentContainerUtils.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/FlashList.test.tsx b/node_modules/@shopify/flash-list/src/__tests__/FlashList.test.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/GridLayoutProviderWithProps.test.ts b/node_modules/@shopify/flash-list/src/__tests__/GridLayoutProviderWithProps.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/MasonryFlashList.test.ts b/node_modules/@shopify/flash-list/src/__tests__/MasonryFlashList.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/PlatformHelper.web.test.ts b/node_modules/@shopify/flash-list/src/__tests__/PlatformHelper.web.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/ViewabilityHelper.test.ts b/node_modules/@shopify/flash-list/src/__tests__/ViewabilityHelper.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/helpers/mountFlashList.tsx b/node_modules/@shopify/flash-list/src/__tests__/helpers/mountFlashList.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/helpers/mountMasonryFlashList.tsx b/node_modules/@shopify/flash-list/src/__tests__/helpers/mountMasonryFlashList.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/__tests__/useBlankAreaTracker.test.tsx b/node_modules/@shopify/flash-list/src/__tests__/useBlankAreaTracker.test.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/benchmark/AutoScrollHelper.ts b/node_modules/@shopify/flash-list/src/benchmark/AutoScrollHelper.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/benchmark/JSFPSMonitor.ts b/node_modules/@shopify/flash-list/src/benchmark/JSFPSMonitor.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/benchmark/roundToDecimalPlaces.ts b/node_modules/@shopify/flash-list/src/benchmark/roundToDecimalPlaces.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/benchmark/useBenchmark.ts b/node_modules/@shopify/flash-list/src/benchmark/useBenchmark.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/benchmark/useBlankAreaTracker.ts b/node_modules/@shopify/flash-list/src/benchmark/useBlankAreaTracker.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/benchmark/useDataMultiplier.ts b/node_modules/@shopify/flash-list/src/benchmark/useDataMultiplier.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/benchmark/useFlatListBenchmark.ts b/node_modules/@shopify/flash-list/src/benchmark/useFlatListBenchmark.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/errors/CustomError.ts b/node_modules/@shopify/flash-list/src/errors/CustomError.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/errors/ExceptionList.ts b/node_modules/@shopify/flash-list/src/errors/ExceptionList.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/errors/Warnings.ts b/node_modules/@shopify/flash-list/src/errors/Warnings.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/index.ts b/node_modules/@shopify/flash-list/src/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutView.tsx b/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutView.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponent.android.ts b/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponent.android.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponent.ios.ts b/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponent.ios.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponent.ts b/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponent.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponentProps.ts b/node_modules/@shopify/flash-list/src/native/auto-layout/AutoLayoutViewNativeComponentProps.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.android.ts b/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.android.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.ios.ts b/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.ios.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.tsx b/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.web.tsx b/node_modules/@shopify/flash-list/src/native/cell-container/CellContainer.web.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.android.ts b/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.android.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.ios.ts b/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.ios.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.ts b/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.web.ts b/node_modules/@shopify/flash-list/src/native/config/PlatformHelper.web.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/utils/AverageWindow.ts b/node_modules/@shopify/flash-list/src/utils/AverageWindow.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/utils/ContentContainerUtils.ts b/node_modules/@shopify/flash-list/src/utils/ContentContainerUtils.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/viewability/ViewToken.ts b/node_modules/@shopify/flash-list/src/viewability/ViewToken.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/viewability/ViewabilityHelper.ts b/node_modules/@shopify/flash-list/src/viewability/ViewabilityHelper.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@shopify/flash-list/src/viewability/ViewabilityManager.ts b/node_modules/@shopify/flash-list/src/viewability/ViewabilityManager.ts
old mode 100644
new mode 100755
